presto-native-execution/presto_cpp/main/runtime-metrics/prometheus-reporter/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+target_sources(presto_server
+               PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/PrometheusStatsReporter.cpp)
```

Comment:
Why is ${CMAKE_CURRENT_SOURCE_DIR} required here ? Haven't seen that used in any other places in the code.

Reply:
`runtime-metrics/PrometheusStatsReporter.cpp` should do.

Reply:
We should simply link with the library below.